log/context.go

```diff
@@ -46,7 +38,7 @@ func WithRequestID(ctx context.Context, requestID string, fields ...LoggableFiel
 // It can be used when there isn't a single, clear, unique id associated with a request (e.g.,
 // a block or tx hash).
 func WithNewRequestID(ctx context.Context, fields ...LoggableField) context.Context {
```

Comment:
I don't think we are doing this correctly - I would expect a request ID being assigned to a request, i.e. something that was triggered by another node via p2p or a client via GRPC. Creating a new block is not a request